pkg/commands/system.go

```diff
@@ -225,7 +227,7 @@ func (i *initCmd) resolveValues() (map[string]interface{}, error) {
 	if i.chartArgs.imagePullPolicy != "" {
 		valuesConfig = append(valuesConfig, fmt.Sprintf("image.pullPolicy=%s", i.chartArgs.imagePullPolicy))
 	}
-	// TODO(wonderflow) values here could give more arguments in command line
+	valuesConfig = append(valuesConfig, i.chartArgs.more...)
```

Comment:
Do you want to put any checks on the format "xxx=xx"?

Reply:
will not, I'm preparing to move it out, and deprecate `vela install`